bitsv/wallet.py

```diff
@@ -135,9 +142,13 @@ class PrivateKey(BaseKey):
                 byte is disregarded. Compression will be used by all new keys.
     :type wif: ``str``
     :raises TypeError: If ``wif`` is not a ``str``.
+    :param apis: A dictionary of {api_name : api_keys} in order of preference for redundancy. (Python 3.6 dicts ordered)
+    :type apis: ``dict`` of ``str`` : ``str``
+    :param network: 'testnet' or 'stn'
```

Comment:
I don't think this is actually set anywhere

Reply:
Woops. I caught this in PrivateKey but not in PrivateKeyTest... hence why a refactoring would be nice at some point to stop with all the duplication of code.